redhat-operators/devconsole/devconsole.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,190 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
This CSV seems like a duplicate of the other one

Reply:
devconsole.v0.1.0.clusterserviceversion.yaml I intended to keep, removed the other one :) That was a mis-copy while copying from https://github.com/redhat-developer/devconsole-operator/tree/master/manifests/devconsole/0.1.0 :)